tools/config_validation/validate_fragment.py

```diff
@@ -0,0 +1,56 @@
+# Validate a YAML fragment against an Envoy API proto3 type.
```

Comment:
This is so, so, so close to the sphinx plugin. ;)

Reply:
Right, @dmitri-d is working on the Sphinx RST plugin side of this at the moment.